.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 coverage
 *.log
-lib/
```

Comment:
the result of compiling should not be checked into source control.

Reply:
i exposed the lib directory because npm/yarn does not seem to run the compile script :thinking: 
![image](https://user-images.githubusercontent.com/44925389/89285106-b2435280-d647-11ea-98ae-4fcf8b825054.png)